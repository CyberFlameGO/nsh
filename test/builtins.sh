source
source test/builtins.sh.source
echo $msg
