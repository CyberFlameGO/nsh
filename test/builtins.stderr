nsh: source: filename argument required
